lib/full-site-editing/block-templates.php

```diff
@@ -165,72 +165,6 @@ function _gutenberg_add_template_part_area_info( $template_info ) {
 	return $template_info;
 }
```

Comment:
I think you should be able to move all the functions from this file into the new `block-template-utils.php` file